.release-notes/4152.md

```diff
@@ -0,0 +1,19 @@
+## Support void* (Pointer[None]) parameters in bare lambdas and functions
+
+Unlike conventional FFI functions, bare lambdas `@{(...)}` and bare functions `fun @bare() => ...` did not support Pointer[None] parameters, Pony's equivalent of `void*`. This is despite the fact that like FFI functions, bare lambdas and functions are strictly intended for use with FFI calls and callbacks.
```

Comment:
Please use consistent `code` formatting here, I see some instances of Pointer[None] and Pointer[A] without it.

Reply:
done